util/cron/test-gasnet-asan.bash

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Test ASAN-compatible configuration with gasnet & ASAN on linux64, running
+# multilocale tests only.
+#
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common-gasnet.bash
+source $CWD/common-asan.bash
+source $CWD/common-localnode-paratest.bash
```

Comment:
do any of these source common.bash? If so, I am fine if you leave it out.

Reply:
Yes `common-gasnet.bash` sources `common.bash`.